progressReport/canvasEmbedding.js

```diff
@@ -0,0 +1,51 @@
+const currentUrl = window.location.href;
+
+
+
+const targetPageUrl = 'https://ucberkeleysandbox.instructure.com/courses/141/pages/concept-map?module_item_id=418';
+const alternateTargetPageUrl = 'https://ucberkeleysandbox.instructure.com/courses/141/pages/concept-map';
+
+
+// Check if the current URL matches the target page URL
+if (currentUrl === targetPageUrl || currentUrl === alternateTargetPageUrl) {
+    const accessToken = '26530~etwvNZ2z3WXDxP9CXDeDNCmKN3MCWEKtDaYrQ7JRVGJ6nkBUcR8V4VURDVMBuLeW';
```

Comment:
Totally forgot to remove this. I have disabled this access token.

Reply:
Please make sure to review all code before committing.  I get it, mistakes happen, but in industry, committing a private key would be a major issue and you'd have a full retro meeting just for this.

Reply:
Noted!